config/billing_rates.yml

```diff
@@ -129,12 +129,14 @@
 - { id: 48e86741-c1ca-8978-bd01-a6e85bb38963, resource_type: IPAddress,                   resource_family: IPv4,                       location: us-east-1,      unit_price: 0.0000000000, billed_by: duration, active_from: 2023-01-01T00:00:00Z }
 - { id: ba5cc13f-dd34-8178-91a6-314dd036bf1e, resource_type: PostgresVCpu,                resource_family: standard-standard,          location: us-west-1,      unit_price: 0.0005518353, billed_by: duration, active_from: 2025-02-17T00:00:00Z }
 - { id: 4130391d-81fd-8178-871c-9511350c0b16, resource_type: PostgresVCpu,                resource_family: standard-standard,          location: us-east-1,      unit_price: 0.0006696429, billed_by: duration, active_from: 2025-02-17T00:00:00Z }
+- { id: 289c6fce-02da-484d-a329-2829c7663f6a, resource_type: PostgresVCpu,                resource_family: standard-standard,          location: us-west-2    ,  unit_price: 0.0009525360, billed_by: duration, active_from: 2025-02-17T00:00:00Z }
```

Comment:
![CleanShot 2025-05-04 at 00 20 50@2x](https://github.com/user-attachments/assets/380958d5-e16e-46fa-8e78-f50146e887df)
extra spaces before the comma